stdlib/tests/crypto/arithmetic_circuit_eval.rs

```diff
@@ -24,7 +24,7 @@ fn arithmetic_circuit_eval_and_execute() {
 
             # Set up the inputs to the arithmetic circuit evaluation op and execute it
             push.NUM_EVAL_ROWS push.NUM_READ_ROWS push.{pointer}
-            arithmetic_circuit_eval
+            circuit_eval
```

Comment:
Looking at this now, I wonder if it should be `eval_circuit`. For an assembly instruction sounds a bit more consistent with other instructions (e.g., leading with the action verb).

Reply:
Agreed!
Went ahead and changed it to use the action verb first